### PR TITLE
Add info for mitigating guideline changes in APIs

### LINKED ISSUE #168: Add recommendation for mitigating guideline changes
Add chapter for guideline changes and how teams should respond to it:

- existing APIs don't have to be changed, but we recommend it
- clients of existing APIs have to cope  with these APIs based on 'old guidelines'
- new APIs have to respect the current guidelines

Furthermore: once an API becomes public external available, it has to be re-reviewed and changed according to current guidelines - for sake of consistency.

### PR DESCRIPTION
**Pull request by fmueller**
Fixes #168

### CODE REVIEW

File: Introduction.md [hunk 1]
```diff
@@ -31,5 +31,15 @@ The purpose of our “RESTful API guidelines” is to define standards to succes
...
 teams can confidently follow and trust them.
+
+In case guidelines are changing, following rules apply:
+
+- existing APIs don't have to be changed, but we recommend it
+- clients of existing APIs have to cope with these APIs based on outdated rules
+- new APIs have to respect the current guidelines
+
+Furthermore you should keep in mind that once an API becomes public external
+available, it has to be re-reviewed and changed according to current
```
Comment (ePaul) [inline Introduction.md]: external → externally. (I'm not sure if the same applies to "public" here too.)

Comment (fmueller) [inline Introduction.md]: Ok, I fixed this typo.

**Comment by fmueller** [comment]
@maxim-tschumak @ePaul Zappr does not read Github approvals, that's why we still need thumbs ;)

**Comment by maxim-tschumak** [comment]
:+1:

**Comment by ePaul** [comment]
:+1:

**Pull request merged**